core/templates/dev/head/player/PlayerServices.js

```diff
@@ -344,6 +387,7 @@ oppia.factory('oppiaPlayerService', [
           sessionId = data.session_id;
           _viewerHasEditingRights = data.can_edit;
           _loadInitialState(successCallback);
+          _loadInitialInformationCardData(successCallback);
```

Comment:
You'll end up calling successCallback twice if you do this. I don't think you need to pass in successCallback...